### PR TITLE
Create reverse_ll_kgroups.cpp

### PR DESCRIPTION
**Pull request by TanishqKhetan**

### CODE REVIEW

**Comment by TanishqKhetan** [comment]
could you please see to this code

File: CPP/reverse_ll_kgroups.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+28
+Pull requests
+129
+Actions
+Projects
+Security
+Insights
+CodeHelp-DSA-Busted-Series/Lecture046 Linked List Day3/reverseInKGroup.cpp
+
+Love Babbar Adding codes for previous Lectures
```
Comment (dikshantrajput) [inline CPP/reverse_ll_kgroups.cpp:24]: Please remove all these stuffs from top and bottom

**Review by dikshantrajput — CHANGES_REQUESTED** [review]

**Review by TanishqKhetan — COMMENTED** [review]
I have updated the code kindly review it

**Comment by TanishqKhetan** [comment]
updated

**Comment by dikshantrajput** [comment]
Have you added your name in the contributors.html file?

**Comment by TanishqKhetan** [comment]
No I have contributed to c++

**Comment by dikshantrajput** [comment]
Please add your name also to the contributors.html file

**Pull request merged**